mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -24,6 +28,22 @@ EndOfMessage
     exit $exitCode
 }
 
+# Get current hardware architecture
+export ARCH=$(uname -m | sed -e 's/aarch64.*/arm64/' -e 's/x86_64.*/amd64/' -e 's/armv.*/arm/')
+if [ "${ARCH}" = "ppc64le" ]; then
+    export ARCH_DEB=ppc64el
+else
+    export ARCH_DEB="${ARCH}"
+fi
+
+# Set the correct default value for docker-compose command
+if [ -f /usr/bin/docker-compose ]; then
```

Comment:
I still think this check is bad. Even tho the code works, because later when the `docker-compose` is installed it will adjust the value of `DOCKER_COMPOSE_CMD`, it is confusing to anyone trying to read the code or debug, because in the amd64 case when `docker-compose` is not installed, `DOCKER_COMPOSE_CMD=pipenv run docker-compose` for part of the script.

This check should be based are the `$ARCH` variable, just like the section that installs docker-compose.

Reply:
Fixed it as you described based on `$ARCH` variable. Close it.